### ISSUE TITLE
Rewrap for VSCode hates triple quoted strings in Python

### ISSUE THREAD

**Issue by Ralithune** [stkb/Rewrap#166]
In VSCode:
```
var = """
This a block of text I want to define with triple quoted strings.
It has multiple lines.
For some reason, Rewrap doesn't like it.
"""

# The above triple quoted string is perfectly legitimate, yet if I wrote a very long single-line comment that goes out past ~100 characters (which is where my file is set at, yes I know it's not technically Google Python standards), pressing Alt+q on this block makes OTHER code in the file re-wrap; not the comment block.  It's really strange.
```
```
var = (
"""
However, if I put parenthesis around the triple quoted block, and then the triple
quotes on the next line (putting them together as in (""" causes the same error),
it works fine.
""")

# If I try to format this single-line comment string using rewrap now, this will re-wrap appropriately.  I think there is some bug with how re-wrap recognizes begin and end of comment blocks with Python.
```

**Comment by Nutznieser** [comment]
I can also agree with this bug for python code.

What also makes no problems is ...
```
var = \
"""
This a block of text I want to define with triple quoted strings.
It has multiple lines.
For some reason, Rewrap doesn't like it.
"""
```
and also without the variable declaration (e.g. for docstrings) you will succeed ...
```
"""
This a block of text I want to define with triple quoted strings.
It has multiple lines.
For some reason, Rewrap doesn't like it.
"""
```

But if you do declaring a variable with a triple quoted string as usually the rewrap process breaks, and not only for the current code piece but also for all code after this.

Beside this Rewrap works great for python code and I hope somebody can fix this bug!

**Comment by Nutznieser** [comment]
Unfortunately this issue is not solved in the new version v1.10.0. It seems that the regular expressions that searches the strings only allows whitespaces before a python multiline string. But in the legitimate examples above the multiline string is bounded to a variable.

**Comment by stkb** [comment]
Closing as a duplicate of #157 ([see my comment there](https://github.com/stkb/Rewrap/issues/157#issuecomment-637635894))
